### DISCUSSION TITLE
How do I reduce multiple notifications for same person?

### DISCUSSION THREAD

**Discussion by vavallee-wp** [skrashevich/double-take#182] [General]
I am having an issue where when a person walks up the driveway, I get multiple notifications of "unknown person" until they get closer, at which time it identifies them and I get 1 or two notifications for that.

Doubletake also seems to be sending excessive requests to my facial recognitions provider. 

What am I doing wrong?

**Comment by skrashevich** [comment]
Use 'masks' option in cameras section to recognise only when person will be in specified area. And/or you can increase min_area.
For reduce count of requests you can also activate 'opencv_face_required' and decrease values in 'attempts' section

**Reply by vavallee-wp** [reply]
Playing with the numbers seems to be helping, thanks
